opm/common/OpmLog/EclipsePRTLog.hpp

```diff
@@ -29,16 +29,21 @@ namespace Opm {
 class EclipsePRTLog : public StreamLog {
 
 public:
-    using StreamLog::StreamLog;
 
     void addTaggedMessage(int64_t messageType, const std::string& messageTag, const std::string& message);
 
     size_t numMessages(int64_t messageType) const;
 
     ~EclipsePRTLog();
 
+    EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
Discovered by Travis: this set of constructors is insufficient compared to that offered previously through inheriting the StreamLog constructors. The two-argument construction taking a string and message mask (with append defaulting to false) is no longer available with this PR, but is used from opm-core/examples/diagnose_relperm.cpp at least. Adding a default for `append` would be sufficient I think.